service/pixelated/resources/login_resource.py

```diff
@@ -151,15 +151,29 @@ def _handle_login(self, request):
         defer.returnValue(user_auth)
 
     def _complete_bootstrap(self, user_auth, request):
-        def log_error(error):
+        def login_error(error, session):
             log.error('Login error during %s services setup: %s \n %s' % (user_auth.username, error.getErrorMessage(), error.getTraceback()))
+            session.login_error()
 
-        def set_session_cookies(_):
-            session = IPixelatedSession(request.getSession())
-            session.user_uuid = user_auth.uuid
+        def set_session_cookies(_, session):
```

Comment:
This method seems to not be doing anything related with cookies (just calls a method that does). Should be rename it to something like `finalize_login`?